cosmos/titan_18888.json

```diff
@@ -0,0 +1,53 @@
+{
+  "rpc": "https://titan-rpc.titanlab.io",
+  "rest": "https://titan-lcd.titanlab.io",
+  "chainId": "titan_18888-1",
+  "chainName": "Titan",
+  "chainSymbolImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/titan_18888/chain.png",
+  "nodeProvider": {
+    "name": "Tokenize",
+    "email": "info@titanlab.io",
+    "website": "https://titanlab.io"
```

Comment:
@InnerPeace080 Could you check your website?

Reply:
Yes. You are right. Ours website did not complete and online yet. Can i just regis chain info first ?

Reply:
@InnerPeace080 It can be hard for users to contact you when a node isn't behaving properly

Reply:
Thank you. So i will create another pull request after website complete. 
Thank you